docker_compose/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.7'
 services:
   api:
-    image: "hashicorpdemoapp/product-api:v0.0.12"
+    image: "hashicorpdemoapp/product-api:v0.0.16"
```

Comment:
should this be 0.17?

Reply:
I think we have a chicken or egg problem here - if the functional tests run, they need an existing image. But we don't have an image until we push it to Docker Hub 🤔